<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>overviewbestellungen_bestellung</class>
 <widget class="QWidget" name="overviewbestellungen_bestellung">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="artikelSuche">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Artikelsuche....</string>
   </property>
  </widget>
  <widget class="QTableView" name="artikelListe">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>571</width>
     <height>101</height>
    </rect>
   </property>
   <property name="placeholderText" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LabelArtikel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Artikel auswählen&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="artikelMenge">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>120</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="hinzufugen">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>120</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hinzufügen</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="warenkorb">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>571</width>
     <height>121</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Warenkorbliste</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelWarenkorb">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>330</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Warenkorb&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bestellen">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>520</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Bestellen</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Menge:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>260</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>KundenID: </string>
   </property>
  </widget>
  <widget class="QLabel" name="ID_Label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>260</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="orders">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Orders</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
